examples/gcp-sample-project/Pulumi.example.yaml

```diff
@@ -1,2 +1,3 @@
 config:
   gcp:project: jvm-lab
+
```

Comment:
(super minor) I'd revert this change, since it doesn't really contribute anything :p